jobs/aws/eks-distro/kubernetes-1-19-release-presubmits.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+presubmits:
+  aws/eks-distro:
+  - name: kubernetes-release-presubmit
```

Comment:
Can you add the release-branch to the job name, for example, `kubernetes-release-1-18/19/20/21-presubmit`?